scene/resources/animation.h

```diff
@@ -90,6 +90,12 @@ class Animation : public Resource {
 		FIND_MODE_EXACT,
 	};
 
+	enum DEFAULT_BEZIER_KEY {
+		DEFAULT_BEZIER_KEY_BEHAVIOR_HORIZONTAL,
+		DEFAULT_BEZIER_KEY_BEHAVIOR_BALANCED_AUTOMATIC,
+		DEFAULT_BEZIER_KEY_BEHAVIOR_MIRRORED_AUTOMATIC
+	};
+
```

Comment:
```suggestion
```

Must be placed in `#ifdef TOOLS_ENABLED` block and use pascal case.

However, I believe you don't actually add this enum, and you can just use `HandleMode`. Although you need to add a "Linear", but I think it can be labeled `HANDLE_MODE_FREE` as "Horizontal" as shown below:

```cpp
EDITOR_SETTING(Variant::INT, PROPERTY_HINT_ENUM, "editors/animation/default_bezier_key_behavior", 0, "Horizontal,Linear,Balanced Auto,Mirrored Auto");
```

Reply:
Yeah this is a good idea. Made sense to have a Linear as an option as well. Done